src/button.js

```diff
@@ -3,20 +3,31 @@ import PropTypes from 'prop-types'
 import { makeStyles } from '@material-ui/core/styles'
 import MUIButton from '@material-ui/core/Button'
 import clsx from 'clsx'
+import Test from './theme/overrides/Test'
 
-const useStyles = makeStyles((theme) => {
-  return {
-    label: {
-      color: 'rgba(0, 0, 0, 0.0) !important',
-    },
-    noSpacing: {
-      padding: theme.spacing(0),
-    },
-  }
-})
+const useStyles = color => {
+  const selectStyle = makeStyles((theme) => {
+    const customColor = Test(color)
```

Comment:
I would stay away from directly importing any theme props to the component as `theme` is already accessible via `makeStyles()`